R/groundhog.library.R

```diff
@@ -61,23 +94,26 @@ groundhog.library <- function(
   install.snowball(pkg, date, include.suggests,
     force.install = force.install,
     force.source = force.source,
-    quiet.install = quiet.install, current.deps = current.deps
+    quiet.install = quiet.install,
+    current.deps = current.deps
   )
 
   # 8.8 Do library() call for pkg_vrs
-  library(pkg, character.only = TRUE)
+  library(pkg, character.only = TRUE, lib.loc = .libPaths()[!.libPaths() %in% orig_lib_paths]) # Exclude from search the default library
```

Comment:
Sounds nice. I'll think about it a bit but it's probably a good idea to prevent issues like both of us encountered. :+1: